App.js

```diff
@@ -1,25 +1,29 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import SvgComponent from "./LogoSvg";
+import LogoSvg from "./LogoSvg";
+import Quotes from "./Quotes";
+import AnimateSvg from './AnimateSvg';
+import AnimateQuotes from './AnimateQuotes';
 
 export default function App() {
-
   return (
-    <View style={styles.container}>
-      <SvgComponent />
+    <View style={styles.window}>
+      <AnimateSvg>
+        <LogoSvg />
+      </AnimateSvg>
+      <AnimateQuotes>
+        <Quotes />
+      </AnimateQuotes>
     </View>
   )
 };
 
 const styles = StyleSheet.create({
-  container: {
+  window: {
     flex: 1,
     justifyContent: 'center',
     paddingHorizontal: 20,
-    backgroundColor: 'black'
+    backgroundColor: 'black',
+    position: 'relative',
```

Comment:
Designated StyleSheet window property as having relative positioning in order to designate children with absolute positioning so that they may occupy the same space in the device window.

Reply:
This is a great use of comments!